pyoverkiz/clients/somfy_client.py

```diff
@@ -0,0 +1,68 @@
+from aiohttp import FormData
+from pyoverkiz.client import OverkizClient
+from pyoverkiz.const import SOMFY_API, SOMFY_CLIENT_ID, SOMFY_CLIENT_SECRET
+from pyoverkiz.exceptions import (
+    BadCredentialsException,
+    SomfyBadCredentialsException,
+    SomfyServiceException,
+)
+
+
+class SomfyEuropeClient(OverkizClient):
```

Comment:
Is this the class where we will store the tokens and handle the refresh?

Reply:
Yes that's the idea.